src/Cimpress.Nancy.Logging/SumologicLogger.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Cimpress.Nancy.Components;
+using log4net;
+
+namespace Cimpress.Nancy.Logging
+{
+    public class SumologicLogger : INancyLogger
+    {
+        private ILog _leLogger;
+
+        private string _applicationName;
+        private string _environmentName;
+        private string _sumoLogicBaseUri;
+
+        public void Configure(IDictionary<string, string> options)
+        {
+            _applicationName = options["ApplicationName"];
```

Comment:
This will throw an exception if not set.
https://msdn.microsoft.com/en-us/library/9tee9ht2(v=vs.110).aspx